docs/Kube-API.md

```diff
@@ -168,7 +168,7 @@ In case of failure to apply the overrides the clusterdeployment conditions will
 Add an annotation with the desired options, the clusterdeployment controller will update the install config yaml with the annotation value.
 Note that this configuration must be applied prior to starting the installation
 ```sh
-$ kubectl annotate agentclusterinstalls.extensions.hive.openshift.io test-cluster -n assisted-installer agent-install.openshift.io/install-config-overrides="{\"controlPlane\":{\"hyperthreading\":\"Disabled\"}}"
+$ kubectl annotate agentclusterinstalls.extensions.hive.openshift.io test-cluster -n assisted-installer agent-install.openshift.io/install-config-overrides="{\"controlPlane\":{\"hyperthreading\":\"Disabled\", \"name\": \"master\"}}"
```

Comment:
This change is not needed, we want users to use the new API to set the hyperthreading.
We should choose another example here, like NetworkType: " OVN-Kubernetes"

Reply:
I don't think that OVN-Kubernetes is a good example since when using IPv6 this is the only option and it is set by default